src/icalendar/prop/__init__.py

```diff
@@ -2061,6 +2849,22 @@ def from_ical(cls, ical):
         return cls(ical)
 
 
+class vUnknown(vText):
+    """This is text but the VALUE parameter is unknown.
+
+    Since :rfc:`7265`, it is important to record if values are unknown.
+    For :rfc:`5545`, we could just assume TEXT.
+    """
+
+    default_value: ClassVar[str] = "UNKNOWN"
+
+    def examples() -> list[vUnknown]:
+        """Examples of vUnknown"""
+        return [vUnknown("Some property text.")]
```

Comment:
```suggestion
    @classmethod
    def examples(cls) -> list[vUnknown]:
        """Examples of vUnknown"""
        return [vUnknown("Some property text.")]
```
All other `examples()` methods use `@classmethod` decorator with `cls` parameter. This one is inconsistent.